install/operator/pkg/syndesis/olm/csv.go

```diff
@@ -207,16 +208,50 @@ func (c *csv) build() (err error) {
 	}
 	c.setVariables()
 
+	//
+	// Ensure the following files can be located
+	//
+	if _, err := os.Stat("pkg"); os.IsNotExist(err) {
+		return fmt.Errorf("This needs to be executed from the syndesis/install/operator directory.")
+	}
```

Comment:
Ideally, if this is important, embedding the data needed in the executable binary might be a better option.

For quick stab at robustness as `pkg` is somewhat widespread, we might go with a marker file, or try to access the required files and error out if they can't be read.

Or if it makes make sense to have a look at the current working directory instead?

```suggestion
	if cwd, err := os.Getwd(); err != nil {
		if strings.HasSuffix(cwd, "/install/operator") == false {
			return fmt.Errorf("This needs to be executed from the syndesis/install/operator directory.")
		}
	} else {
		return err
	}
```